docs/fleet/configure-a-fleet.md

```diff
@@ -26,7 +26,7 @@ Once you are confident that your configuration change works as expected, you can
 If you attempt to delete a fragment that is currently deployed to a machine, you will receive an error message advising that the fragment is in use, but you can still delete the fragment if desired.
 You can see the number of machines using your fragment from the [fragments page](https://app.viam.com/fragments) in the Viam app.
 
-Fragments are private to each organization by default (except for the `viam-dev` organization).
+Fragments are private to each organization by default.
```

Comment:
It feels weird to me to call out `viam-dev` here, idk if a security risk. If anybody thinks it's necessary I can leave?

Reply:
yeah that is weird